docs/upgrade/UPGRADE-v7.1.0.md

```diff
@@ -0,0 +1,173 @@
+    ```diff
+    {% for domain in domains -%}
+    -   grunt.registerTask('frontend{{ domain.id }}', ['webfont:frontend', 'sprite:frontend', 'less:frontend{{ domain.id }}', 'legacssy:frontend{{ domain.id }}', 'less:wysiwyg{{ domain.id }}'], 'postcss');
+    +   grunt.registerTask('frontend{{ domain.id }}', ['webfont:frontend', 'sprite:frontend', 'less:frontend{{ domain.id }}', 'less:print{{ domain.id }}', 'legacssy:frontend{{ domain.id }}', 'less:wysiwyg{{ domain.id }}'], 'postcss');
+    {% endfor -%}
+    grunt.registerTask('admin', ['sprite:admin', 'webfont:admin', 'less:admin', 'legacssy:admin' ]);
+
+    {% for domain in domains -%}
+    -   grunt.registerTask('frontendLess{{ domain.id }}', ['less:frontend{{ domain.id }}', 'legacssy:frontend{{ domain.id }}', 'less:wysiwyg{{ domain.id }}']);
+    +   grunt.registerTask('frontendLess{{ domain.id }}', ['less:frontend{{ domain.id }}', 'legacssy:frontend{{ domain.id }}', 'less:print{{ domain.id }}', 'less:wysiwyg{{ domain.id }}']);
+    {% endfor -%}
+    ```
+    - attach generated print CSS file in document header. In order to do that you have to modify `src/Shopsys/ShopBundle/Resources/views/Front/Layout/base.html.twig` as is shown below.
+    ```diff
+    <link rel="stylesheet" type="text/css" href="{{ asset('assets/frontend/styles/index' ~ getDomain().id ~ '_' ~ getCssVersion() ~ '.css') }}" media="screen, projection">
+    <!--[if lte IE 8 ]>
+        <link rel="stylesheet" type="text/css" href="{{ asset('assets/frontend/styles/index' ~ getDomain().id ~ '_' ~ getCssVersion() ~ '-ie8.css') }}" media="screen, projection">
+    <![endif]-->
+
+    + <link rel="stylesheet" type="text/css" href="{{ asset('assets/frontend/styles/print' ~ getDomain().id ~ '_' ~ getCssVersion() ~ '.css') }}" media="print">
+    ```
+    - generate new `Grtuntfile.js` by running command `php phing gruntfile`
+    - generate new CSS files by running command `php phing grunt`
+- *(low priority)* add custom message for unique e-mail validation in `/src/Shopsys/ShopBundle/Form/Front/Registration/RegistrationFormType.php` ([#885](https://github.com/shopsys/shopsys/pull/885))
+    ```diff
+    -                    new UniqueEmail(),
+    +                    new UniqueEmail(['message' => 'This e-mail is already registered']),
+    ```
+    - dump translations via `php phing dump-translations`
+    - fix `CustomerRegistrationCest::testAlreadyUsedEmail` acceptation test
+        ```diff
+        -        $registrationPage->seeEmailError('Email no-reply@shopsys.com is already registered');
+        +        $registrationPage->seeEmailError('This e-mail is already registered');
+        ```
+- *(low priority)* to improve your deployment process and avoid possible Redis cache problems during deployment, include `build-version` into your builds ([#886](https://github.com/shopsys/shopsys/pull/886))
```

Comment:
should not this rather be in "Infrastructure" or "Configuration" section? :thinking:

Reply:
moved